portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,21 +40,28 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json;");
     response.getWriter().println(convertToJsonUsingGson(comments));
+    
   }
 
   private String convertToJsonUsingGson(ArrayList<String> jsonData) {
     return gson.toJson(jsonData);
   }
   
-  // Get the comments from the user and display them
+  // Get the comments from the user and add them to datastore
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String input = getParameter(request, "text-input", "");
     comments.add(input);
 
-    
-    response.setContentType("text/html");
-    response.getWriter().println(comments);
+    Entity commentEntity = new Entity("Comments");
```

Comment:
You should use Java constants for `"Comments"` and other strings here that you reference in multiple places. This prevents errors related to typos and makes it easier to maintain by having a single place where the string needs to be updated if it needs to be changed.

Reply:
Ok will do